gulpfile.js

```diff
@@ -0,0 +1,82 @@
+var gulp = require('gulp');
+var gutil = require('gulp-util');
+var rename = require('gulp-rename');
+var path = require('path');
+var join = path.join;
+
+
+gulp.task('default', ['watch']);
+gulp.task('build', ['styles']);
+
+var scssFiles = {
+	'wagtail/wagtailadmin/static': [
+		'wagtailadmin/scss/core.scss',
+		'wagtailadmin/scss/layouts/login.scss',
+		'wagtailadmin/scss/layouts/home.scss',
+		'wagtailadmin/scss/layouts/page-editor.scss',
+		'wagtailadmin/scss/layouts/preview.scss',
+		'wagtailadmin/scss/panels/rich-text.scss',
+		'wagtailadmin/scss/userbar.scss',
+		'wagtailadmin/scss/normalize.scss',
+		'wagtailadmin/scss/userbar_embed.scss',
+	],
+	'wagtail/wagtailimages/static': [
+		'wagtailimages/scss/add-multiple.scss',
+		'wagtailimages/scss/focal-point-chooser.scss',
+	],
+	'wagtail/wagtailusers/static': [
+	],
+};
+
+
+/*
+ * Watch - Watch files, trigger tasks when they are modified
+ */
+gulp.task('watch', ['build'], function () {
+	for (var appPath in scssFiles) {
+		gulp.watch(join(appPath, '*/scss/**'), ['styles']);
+	}
+});
+
+
+/*
+ * SASS - Compile and move sass
+**/
+
+gulp.task('styles', function () {
+	var sass = require('gulp-sass');
+	var pixrem = require('gulp-pixrem');
+	var autoprefixer = require('gulp-autoprefixer');
```

Comment:
Hey, I've noticed sometimes it can be slower to run a task if the modules are required within the task itself... I'd suggest moving the `require()` statements to the head of the file.